src/views/blocks/rsvp/content.php

```diff
@@ -13,15 +13,15 @@
  * @version 4.9
  *
  */
-
+$going = ! empty( $_GET[ 'going' ] ) ? $_GET[ 'going' ] : '';
```

Comment:
Should this also use sanitize_text_field() even if $going is esc_html() in the template itself?

Reply:
Is the intention of `going` to be a 1 in these cases? Should this be set accordingly?

Reply:
My bad it's `yes` or empty, so yeah let's just go ahead and add sanitize_text_field() here anyway

Reply:
OK, sanitized - worth noting that that value is via a checkbox, so not _truly_ user input. Still, worth sanitizing anything received through `$_GET`